### PR TITLE
Fix typo of risc-v readme

### PR DESCRIPTION
**Pull request by rednoah91**

### CODE REVIEW

**Review by hanhanW — APPROVED** [review]

File: docs/developers/get_started/getting_started_riscv_cmake.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ $ cmake -G Ninja -B ../iree-build-riscv/ \
   -DIREE_BUILD_COMPILER=OFF \
   -DIREE_ENABLE_MLIR=OFF \
   -DIREE_BUILD_SAMPLES=ON \
-  -DRISCV_TOOLCHAIN_ROOT=${RISCV_TOOLCHAIN_ROOT}
+  -DRISCV_TOOLCHAIN_ROOT=${RISCV_TOOLCHAIN_ROOT} \
```
Comment (ScottTodd) [inline docs/developers/get_started/getting_started_riscv_cmake.md:82]: this should also be fixed in this doc: https://github.com/google/iree/blob/main/docs/website/docs/building-from-source/riscv.md#risc-v-64-bit-linux-target

File: docs/developers/get_started/getting_started_riscv_cmake.md [hunk 2]
```diff
@@ -207,7 +207,7 @@ $ ../iree-build-host/install/bin/iree-translate \
 Then run on the RISC-V QEMU:
 
 ```shell
-$ ${QEMU_BIN}} \
+$ ${QEMU_BIN} \
```
Comment (ScottTodd) [inline docs/developers/get_started/getting_started_riscv_cmake.md:210]: same here: https://github.com/google/iree/blob/main/docs/website/docs/building-from-source/riscv.md#optional-configuration

**Comment by ScottTodd** [comment]
btw Hanhan, for doc-only changes, I just skip checks and merge without waiting :P
I think that requires admin though

**Comment by hanhanW** [comment]
> btw Hanhan, for doc-only changes, I just skip checks and merge without waiting :P
> I think that requires admin though

yeah, I don't have admin permission. I'm fine with it because we can turn on auto-merge. 📦 

I disable the auto-merge, @rednoah91 could you also fix the typos that Scott pointed out?

File: docs/developers/get_started/getting_started_riscv_cmake.md (hunk 1 above)
Comment (rednoah91) [reply docs/developers/get_started/getting_started_riscv_cmake.md:82]: OK

File: docs/developers/get_started/getting_started_riscv_cmake.md (hunk 2 above)
Comment (rednoah91) [reply docs/developers/get_started/getting_started_riscv_cmake.md:210]: OK

**Comment by rednoah91** [comment]
> > btw Hanhan, for doc-only changes, I just skip checks and merge without waiting :P
> > I think that requires admin though
> 
> yeah, I don't have admin permission. I'm fine with it because we can turn on auto-merge. 📦
> 
> I disable the auto-merge, @rednoah91 could you also fix the typos that Scott pointed out?

Done. Please check it. Thanks.

**Review by ScottTodd — APPROVED** [review]

**Comment by ScottTodd** [comment]
Thanks!

**Pull request merged**